README.md

```diff
@@ -72,11 +72,25 @@ To publish your application to Amazon S3 and Amazon CloudFront:
 
  Done!
 
+ ## Enabling federated sign-in
+
+Federated sign-in controls for Google, Facebook and Amazon are provided in the user interface by default; however, the client ids for these providers are not valid.  Dummy values are provided in the **federated** object within index.js so that the user interface controls appear.  You can remove any of the controls by deleting the appropriate keys from the **federated** object.
+
+In order to enable federated sign-in using these providers, please [check the documentation](https://aws.github.io/aws-amplify/media/federated_identity_setup).
```

Comment:
Thinking from the customer point of view, it will be very difficult to make it work. Because you need to configure the IdentityPool to have Google, Facebook, Amazon as an Auth provider. In case you have a facebook appid or google client id is simple to do it using awsmobile-cli. I would leave empty by default and I would add instructions for each platform. I think it would be valuable for customer having that on the starter.